core/config/docs/chains-solana.toml

```diff
@@ -39,6 +39,38 @@ BlockHistoryPollPeriod = '5s' # Default
 # ComputeUnitLimitDefault is the compute units limit applied to transactions unless overriden during the txm enqueue
 ComputeUnitLimitDefault = 200_000 # Default
 
+[Solana.MultiNode]
+# Enabled enables the multinode feature.
+Enabled = false # Default
+# PollFailureThreshold is the number of consecutive poll failures before a node is considered unhealthy.
+PollFailureThreshold = 5 # Default
+# PollInterval is the rate to poll for node health.
+PollInterval = '10s' # Default
+# SelectionMode is the method used to select the next best node to use.
+SelectionMode = 'PriorityLevel' # Default
+# SyncThreshold is the number of blocks behind the best node that a node can be before it is considered out of sync.
+SyncThreshold = 5 # Default
+# NodeIsSyncingEnabled enables the feature to avoid sending transactions to nodes that are syncing. Not relavant for Solana.
+NodeIsSyncingEnabled = false # Default
+# LeaseDuration is the max duration a node can be leased for.
```

Comment:
Can we please do another pass on these docs? Phrases like this are skirting the documentation requirement by saying nothing in particular, and in this case we already have good EVM examples to draw on.

Reply:
I merged to unblock other PRs, but please circle back to this.

Reply:
Yeah sounds good, I'll create a ticket for later!